api/v1alpha1/openapi.yaml

```diff
@@ -256,6 +263,13 @@ paths:
         - agent
       description: list agents
       operationId: listAgents
+      parameters:
+        - name: with_example
+          in: query
+          description: Returning the list of agents along with the default report
```

Comment:
this should be the description of the parameter not of the action like `control whatever the default report should be added to the result`